Configuration/TSconfig/page.ts

```diff
@@ -0,0 +1,299 @@
+# include contentWizard settings
```

Comment:
Please name the file multicolumn.ts as there is not page but mutlicolumn configuration.